newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -253,6 +255,12 @@ export default class SearchPanel extends PureComponent<Props, State> {
               >
                 <ChevronRight />
               </IconButton>
+              <FlatButton
+                key="close"
+                label={<Trans>Close</Trans>}
+                primary={false}
+                onClick={()=>{this.props.onCloseSearchPanel()}}
```

Comment:
You can remove `this.props.`

But for that you need add the props `onCloseSearchPanel `in the destructuring drops at line 120 of this file.

Reply:
Take a look on `onGoToPreviousSearchResult`, how it's define at line 120, and how it's used :)